tests/test_cli_compile.py

```diff
@@ -1656,3 +1656,118 @@ def test_triple_equal_pinned_dependency_is_used(
     assert out.exit_code == 0, out
     for line in out_expected_content:
         assert line in out.stderr
+
+
+@pytest.mark.network
+@pytest.mark.parametrize(
+    ("fname", "content"),
+    (
+        # setuptools
```

Comment:
Could we use `pytest.param(..., id='..')` instead of comments? That improves `pytest -v` output.

Befure:

```
tests/test_cli_compile.py::test_input_formats[setup.cfg-\n                [metadata]\n                name = sample_lib\n                author = Vincent Driessen\n                author_email = me@nvie.com\n\n                [options]\n                packages = find:\n                install_requires =\n                    small-fake-a==0.1\n                    small-fake-b==0.2\n\n                [options.extras_require]\n                dev =\n                    small-fake-c==0.3\n                    small-fake-d==0.4\n                test =\n                    small-fake-e==0.5\n                    small-fake-f==0.6\n            ] PASSED [ 25%]
tests/test_cli_compile.py::test_input_formats[setup.py-\n                from setuptools import setup\n\n                setup(\n                    name="sample_lib",\n                    version=0.1,\n                    install_requires=["small-fake-a==0.1", "small-fake-b==0.2"],\n                    extras_require={\n                        "dev": ["small-fake-c==0.3", "small-fake-d==0.4"],\n                        "test": ["small-fake-e==0.5", "small-fake-f==0.6"],\n                    },\n                )\n            ] PASSED [ 50%]
tests/test_cli_compile.py::test_input_formats[pyproject.toml-\n                [build-system]\n                requires = ["flit_core >=2,<4"]\n                build-backend = "flit_core.buildapi"\n\n                [tool.flit.metadata]\n                module = "sample_lib"\n                author = "Vincent Driessen"\n                author-email = "me@nvie.com"\n\n                requires = ["small-fake-a==0.1", "small-fake-b==0.2"]\n\n                [tool.flit.metadata.requires-extra]\n                dev  = ["small-fake-c==0.3", "small-fake-d==0.4"]\n                test = ["small-fake-e==0.5", "small-fake-f==0.6"]\n            ] PASSED [ 75%]
tests/test_cli_compile.py::test_input_formats[pyproject.toml-\n                [build-system]\n                requires = ["poetry_core>=1.0.0"]\n                build-backend = "poetry.core.masonry.api"\n\n                [tool.poetry]\n                name = "sample_lib"\n                version = "0.1.0"\n                description = ""\n                authors = ["Vincent Driessen <me@nvie.com>"]\n\n                [tool.poetry.dependencies]\n                python = "*"\n                small-fake-a = "0.1"\n                small-fake-b = "0.2"\n\n                small-fake-c = "0.3"\n                small-fake-d = "0.4"\n                small-fake-e = "0.5"\n                small-fake-f = "0.6"\n\n                [tool.poetry.extras]\n                dev  = ["small-fake-c", "small-fake-d"]\n                test = ["small-fake-e", "small-fake-f"]\n            ] PASSED [100%]
```

After:

```
tests/test_cli_compile.py::test_input_formats[setup.cfg] PASSED                                                                                                                                      [ 25%]
tests/test_cli_compile.py::test_input_formats[setup.py] PASSED
tests/test_cli_compile.py::test_input_formats[flit] PASSED
tests/test_cli_compile.py::test_input_formats[poetry] PASSED
```

Reply:
Yep, great idea. Fixed 👍🏿

Reply:
FYI another nice way to do this would be to add `ids=('setup.cfg', 'setup.py')`. Sometimes it's easier than embedding the ids into each param. But of course, it depends on the situation.

Reply:
Good to know, thank you 👍🏿 I think in this situation it looks nice to have it per case.